daemon/cmd/daemon_main.go

```diff
@@ -1436,6 +1437,22 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 	return restAPI
 }
 
+func checkifKernelSockmapOptionsConfigured() {
+	if option.Config.SockopsEnable {
+		found := false
+		if h := probes.NewProbeManager().GetHelpers("sock_ops"); h != nil {
+			if _, ok := h["bpf_map_lookup_elem"]; ok {
+				found = true
+			}
+		}
+		if !found {
+			msg := "BPF Sock_ops not supported by kernel."
+			log.Warn(msg + " Disabling the feature.")
```

Comment:
We're not handling a `strict` mode as in NodePort, so we can probably get rid of the `msg` variable.

Reply:
will fix this.